code/datums/objective/individual_objective/technomancers.dm

```diff
@@ -0,0 +1,102 @@
+/datum/individual_objective/disturbance
+	name = "Disturbance"
+	req_department = list(DEPARTMENT_ENGINEERING)
+	units_requested = 3 MINUTES//work, change it to 10
+	based_time = TRUE
+	var/area/target_area
+	var/timer	
+
+/datum/individual_objective/disturbance/assign()
+	..()
+	target_area = random_ship_area()
+	desc = "Something in bluespace tries mess with ship systems. You need to go to [target_area] and power it down by APC \
+	for [unit2time(units_requested)] minutes to lower bluespace interference, before worst will happen."
```

Comment:
Bad wording.
"Something in bluespace tries mess with ship systems. You need to go to [target_area] and power down its APC \
	for [unit2time(units_requested)] minutes to lower bluespace interference, before something worse will happen."
It would be extremely cool if failing this objective would give storyteller some points.